test/fuzz/lua/test_engine.lua

```diff
@@ -1802,6 +1822,82 @@ local function run_test(num_workers, test_duration, test_dir,
     os.exit(exit_code)
 end
 
+local space_id_func
+
+local function TestOneInput(buf)
+    -- Suppress logging by the test and tarantool, libFuzzer
+    -- output is more informative.
+    log.cfg({ level = "warn" })
+    local verbose_mode = false
+    local test_dir = DEFAULT_TEST_DIR
+    local fdp = luzer.FuzzedDataProvider(buf)
+    local math_random = function(fdp)
+        assert(fdp ~= nil)
+        return function(n1, n2)
+            if not n2 then
+                if not n1 then
+                    return fdp:consume_number(0, 1)
+                end
+                n2 = n1
+                n1 = 1
+            end
+            return fdp:consume_integer(n1, n2)
+        end
+    end
+    math.random = math_random(fdp)
+    local seed = fdp:consume_integer(1, 1000)
```

Comment:
It would be nice to set the `seed` from the given arguments as well.

Reply:
You can do it in the same way as for any other libFuzzer test:
`test/fuzz/lua/test_engine.lua --seed=$RANDOM`
This seed is internal and used for initialization of the `math.random()`.